### ISSUE TITLE
[idea] Approval ratings and major election outcomes

### ISSUE THREAD

**Issue by rgrp** [datasets/registry#74]
Not sure about this a core dataset - interested in thoughts.

**Comment by ekoner** [comment]
I'm guessing this is global, not just US?

**Comment by rgrp** [comment]
@ekoner that's right - though you may limit to countries with good approval ratings data.

**Comment by rgrp** [comment]
@ekoner are you up for researching this one?

**Comment by rgrp** [comment]
@ekoner how are you doing here?

**Comment by ekoner** [comment]
@rgrp can't take this on at the moment, too many balls in the air so to speak.
